app/components/language-select.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  config: Ember.inject.service(),
+  i18n: Ember.inject.service(),
+  languageOptions: Ember.computed(function() {
+    return Object.keys(this.get('languageMap'));
+  }),
+  onFinish: null,
+  languageMap: Ember.computed(() => {
+    return {
+      'English': 'en',
+      'French': 'fr'
+    };
+  }),
+  selectedLanguage: null,
+
+  _setUserLanguageChoice(language) {
```

Comment:
Hmmm, I thought the record being written was in fact being used, specifically by the `afterModel` hook in `app/routes/application.js`.  My intent here in `_setUserLanguageChoice` was to write the user's language preference to the database, and then if the user logs out and later logs back in, their language choice is retrieved from the DB in the above-mentioned `afterModel` hook.  I tested it in the browser and the app's behavior is what I expected, but am I confused about how the behavior works under the hood?